backend/src/schema/resolvers/login.spec.js

```diff
@@ -0,0 +1,80 @@
+const { userData } = require("../../data");
+
+const { gql } = require("apollo-server");
+const { createTestClient } = require("apollo-server-testing");
+
+const { constructTestServer } = require("../../utils/__utils");
+
+const POST_LOGIN = gql`
+  mutation login($username: String) {
+    login(username: $username) {
+      isLoggedIn
+      token
+    }
+  }
+`;
+
+const POST_SIGNUP = gql`
+  mutation signup($username: String) {
+    signup(username: $username) {
+      username
+      isLoggedIn
+      token
+    }
+  }
+`;
+
+let mutate;
+
+beforeAll(() => {
+  const { testServer } = constructTestServer();
+  mutate = createTestClient(testServer).mutate;
+});
+
+describe("Mutations", () => {
+  it("receiving token on login", async () => {
+    let res = await mutate({
+      mutation: POST_LOGIN,
+      variables: {
+        username: userData[0].username
+      }
+    });
+    expect(res.data.login.isLoggedIn).toBeTruthy();
```

Comment:
```suggestion
    expect(res.data.login.isLoggedIn).toBe(expect.any(String));
```
would be a little more specific.

Reply:
this is also about ```isLoggedIn``` being useless. Changed it to:
```expect(res.data.login.token).toStrictEqual(expect.any(String))```

Reply:
Right, `isLoggedIn` wasn't the token but some boolean. I meant `token`.